### PR TITLE
VsIG-MySQL-CRUD-1

### PR DESCRIPTION
**Pull request by VsIG-official**

### CODE REVIEW

File: mysql/connector.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+        Example:
+            result = MySQL().create_table('users', [
+                'id INT AUTO_INCREMENT PRIMARY KEY',
+                'name VARCHAR(45)',
+                'age INT',
+                'gender VARCHAR(45)',
+                'nationality VARCHAR(45)'
+            ])
+            assert result == True
```
Comment (mezgoodle) [inline mysql/connector.py:40]: That is not necessary

Comment (mezgoodle) [reply mysql/connector.py:40]: I am talking about `assert result == True`

File: mysql/example.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+data = database.get('SELECT * from users')
+print(data)
+
+result = database.update('users', "name = 'Valentyn'", 'id = 2')
+assert result is True
+
+result = database.delete('users', 'id = 5')
+assert result is True
+
+database.close()
```
Comment (mezgoodle) [inline mysql/example.py:33]: Add blank line at the end of file

**Review by mezgoodle — CHANGES_REQUESTED** [review]
Just some changes

**Review by mezgoodle — APPROVED** [review]

**Review by Dmytrenko-Roman — APPROVED** [review]

**Pull request merged**